app/controllers/katello/api/v2/host_subscriptions_controller.rb

```diff
@@ -56,13 +56,6 @@ def auto_attach
       respond_for_index(:collection => index_response, :template => "index")
     end
 
-    api :GET, "/hosts/:host_id/subscriptions/events", N_("List subscription events for the host"), :deprecated => true
```

Comment:
We should remove the route for this too

Reply:
took me a minute to find, but that's done now :)